src/Lykke.Service.EthereumClassicApi/Filters/ExceptionFilterAttribute.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Lykke.Common.Api.Contract.Responses;
+using Lykke.Service.EthereumClassicApi.Common.Exceptions;
+using Microsoft.AspNetCore.Mvc.Filters;
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Text;
+
+namespace Lykke.Service.EthereumClassicApi.Filters
+{
+    [AttributeUsage(AttributeTargets.Method)]
+    public class ExceptionFilterAttribute : Attribute, IExceptionFilter
```

Comment:
There is https://github.com/LykkeCity/Lykke.Common.ApiLibrary/blob/master/src/Lykke.Common.ApiLibrary/Middleware/GlobalErrorHandlerMiddleware.cs

Reply:
It added, to configure http status codes for different exception types